src/Atom.jl

```diff
@@ -38,9 +37,8 @@ function __init__()
     nothing
   end
 
-  Requires.@require WebIO="0f1e0344-ec1d-5b48-a673-e5cf874b6c29" begin
-    include("display/webio.jl")
-  end
+  @require WebIO="0f1e0344-ec1d-5b48-a673-e5cf874b6c29" include("display/webio.jl")
```

Comment:
I'm not sure why `webio.jl` is better to be loaded by `@require`. Maybe because building WebIO sometimes fails ?